targets/TARGET_ARM_SSG/TARGET_MPS2/TARGET_MPS2_M0/device/TOOLCHAIN_ARM_STD/MPS2.sct

```diff
@@ -42,19 +58,29 @@
 #  endif
 #endif
 
-#define Stack_Size MBED_CONF_TARGET_BOOT_STACK_SIZE
-
-LR_IROM1 0x00000000 0x00400000  {    ; load region size_region
-  ER_IROM1 0x00000000 0x00400000  {  ; load address = execution address
-   *.o (RESET, +First)
-   *(InRoot$$Sections)
-   *(+RO)
-  }
-  ; Total: 64 vectors = 256 bytes (0x100) to be reserved in RAM
-  RW_IRAM1 (0x20000000+0x100) (0x400000-0x100-Stack_Size)  {  ; RW data
-   *(+RW +ZI)
-  }
-  ARM_LIB_STACK 0x20000000+0x400000 EMPTY -Stack_Size { ; Stack region growing down
-  }
-}
+; 8_byte_aligned(48 vect * 4 bytes) =  8_byte_aligned(0xC0) = 0xC0
+#define VECTOR_SIZE                 0x100
+
+#define RAM_FIXED_SIZE              (MBED_CONF_TARGET_BOOT_STACK_SIZE + VECTOR_SIZE)
+
+#define MBED_RAM1_START (MBED_RAM_START + VECTOR_SIZE)
+#define MBED_RAM1_SIZE  (MBED_RAM_SIZE - VECTOR_SIZE)
```

Comment:
This does not match was there before as the size of :
`RW_IRAM1 (0x20000000+0x100) (0x400000-0x100-Stack_Size) {`

Now:
`RW_IRAM1 MBED_RAM1_START MBED_RAM1_SIZE {`

Should it be:
`RW_IRAM1 MBED_RAM1_START (MBED_RAM1_SIZE-MBED_CONF_TARGET_BOOT_STACK_SIZE)`

Is there a reason you changed it? Did you find it was incorrect? Can you double check with the MUSCA team?

Reply:
This is a mistake, I was trying to preserve the hard coded values. What seems to have changed is that in the original `MBED_RAM1_SIZE = MBED_RAM_SIZE - MBED_CONF_TARGET_BOOT_STACK_SIZE - VECTOR_SIZE` (which is equal to what you've suggested), although I thought RAM1 includes the fixed boot stack.

Reply:
Some boards have [subtleties that are not always documented](https://github.com/ARMmbed/mbed-os/issues/14273). So it is probably better to have a clear reason to change them if we are changing them.

Reply:
How is that region size calculated for the GCC_ARM toolchain linker file?